core/src/test/scala/org/apache/spark/status/api/v1/SimpleDateParamTest.scala

```diff
@@ -21,6 +21,11 @@ import org.scalatest.{Matchers, FunSuite}
 class SimpleDateParamTest extends FunSuite with Matchers {
 
   test("date parsing") {
+    import java.util.{Locale, TimeZone}
```

Comment:
@srowen i think i get it. see https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/status/api/v1/SimpleDateParam.scala#L51
`SimpleDateParam` use `SimpleDateFormat` to parse the date string and SimpleDateFormat is sensitive for time zone and we can set the time zone for simpledateformat by 
`SimpleDateFormat.setTimeZone" 

http://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html